### PR TITLE
Fix for parsing issue

### PR DESCRIPTION
**Pull request by dmielaikaite**

### CODE REVIEW

File: AAAOps/Federation/create_allow-list.sh [hunk 1]
```diff
@@ -81,12 +81,13 @@ echo "* redirect cms-xrd-transit.cern.ch+:1213" >> $FEDINFO/out/list_eu.allow
 #cat $FEDINFO/in/prod.txt | cut -d : -f1 | sort -u | awk -F. '{if ($NF == "uk" || $NF == "fr" || $NF == "it" || $(NF-1) == "cern" ) print $(NF-2)"."$(NF-1)"."$NF; else if ( $(NF-1) == "vanderbilt" ) print $(NF-3)"."$(NF-2)"."$(NF-1)"."$NF; else if ( $(NF-1) == "mit" ) print $(NF-2)"."$(NF-1)"."$NF;  else print $(NF-1)"."$NF}' | sort -u > $FEDINFO/in/prod_domain.txt
 
 #Quick fix for "[" character in prod.txt 
-cat $FEDINFO/in/prod.txt | awk '{ if ($1 == "[") print "Unknown.Host"; else print $1;}' > $FEDINFO/in/tmp
+cat $FEDINFO/in/prod.txt | awk '{ if ($1 == "[2804") print "Unknown.Host"; else print $1;}' > $FEDINFO/in/tmp
```
Comment (zvada) [inline AAAOps/Federation/create_allow-list.sh]: Ugh, `"[2804"` is a nasty fix, could you update it with some sort of regex form perhaps? We might have other situations that match `[` but won't satisfy `[2804` in all of the cases.

File: AAAOps/Federation/create_allow-list.sh [hunk 2]
```diff
@@ -81,12 +81,13 @@ echo "* redirect cms-xrd-transit.cern.ch+:1213" >> $FEDINFO/out/list_eu.allow
...
+cat $FEDINFO/in/prod.txt > $FEDINFO/in/tmp
```
Comment (zvada) [inline AAAOps/Federation/create_allow-list.sh]: Don't you override `$FEDINFO/in/tmp` file again which you just parsed/filetered in the previous step? Is this line necessary at all?

File: AAAOps/Federation/create_allow-list.sh (hunk 1 above)
Comment (dmielaikaite) [reply AAAOps/Federation/create_allow-list.sh]: my fault.. the line was pushed by mistake. i have removed it.

File: AAAOps/Federation/create_allow-list.sh (hunk 2 above)
Comment (dmielaikaite) [reply AAAOps/Federation/create_allow-list.sh]: my fault.. the line was pushed by mistake. I have removed it.

File: AAAOps/Federation/create_allow-list.sh (hunk 1 above)
Comment (dmielaikaite) [reply AAAOps/Federation/create_allow-list.sh]: thanks for pointing this out. I have added a regex.

**Review by zvada — APPROVED** [review]
LGTM

**Pull request merged**